kmk/quickpin/pro_micro/liatris.py

```diff
@@ -0,0 +1,46 @@
+import board
```

Comment:
This entire file seems to be out of place in this PR. May be worth opening a separate PR for this.

Reply:
The Liatris controller is a controller designed by splitkb.com so it kind of makes sence to have their keyboard and the quickpin for their controller together. But I could also understand having them separate.
You pose the question 'it may be worth'. If you or someone says 'it should be', then I'll be happy to make a separate PR.

Reply:
I generally agree with splitting contributions into "atomic" commits. A seperate commit in the same PR would be nicer, but in this specific case I don't care too much either way.